### DISCUSSION TITLE
Rebooted server, now cronicle starts, but hangs (in browser) on "Waiting for master server"

### DISCUSSION THREAD

**Discussion by jbwaclawski** [jhuckaby/Cronicle#507] [Q&A] [answered]
There's only one server involved so it's not like it's waiting for another server to do something.  I can't find any usable information in the logs (application and system).

Any ideas on how I can begin to troubleshoot this?

**Comment by jhuckaby** [comment, marked as answer]
Your server's hostname likely changed on reboot.  Cronicle is designed with the assumption that the master server hostname is static and never changes.  If it does, you will have to either change it back, or edit the database to update the hostname.

Try the solution in this issue: https://github.com/jhuckaby/Cronicle/issues/474

**Reply by jbwaclawski** [reply]
So not sure if you saw this before the edit, but I felt necessary to just wipe the previous comment and rewrite it.

Initially the suggestions made in the post you linked didn't work for me, but that's because I didn't take the time to really understand what was going on.  After I re-read the instructions I moved the original data back (thankfully I didn't entirely nuke, just moved the old folder) and re-ran the steps, this time paying closer attention to what I entered for the hostname it was expecting and the group regex it required to match.

All is good now!  Back online, things are working. Thanks for your help!

**Reply by jhuckaby** [reply]
Ugh, I'm very sorry you had to do that.  I'm not sure what happened exactly, but I will say this.  The entire server group / master server system has been COMPLETELY rewritten from the ground up in v2.0, and will never have this problem.  v2 is scheduled for release this year (2022).
